pandoc.cabal

```diff
@@ -426,6 +426,7 @@ library
                  doctemplates >= 0.8 && < 0.9,
                  network-uri >= 2.6 && < 2.7,
                  network >= 2.6,
+                 connection >= 0.3.1,
                  http-client >= 0.4.30 && < 0.7,
                  http-client-tls >= 0.2.4 && < 0.4,
```

Comment:
Is it now possible to remove the dependency on `http-client-tls`, if we're using connection now?
At first I was not too happy about bringing in a new dependency, but since `http-client-tls` depends on `connection` this would be a net reduction!

Reply:
I also wonder whether `connection` would allow simplifying the code for `openURL`?

Reply:
I think we still need http-client-tls to define the TLSSettings in Http client manager. 
And I don't see how to improve the existing code with the connection package (which is very low level , I think). But I am really new to Haskell so I may be missing something.